### DISCUSSION TITLE
glTF ophans entities

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [google/filament#7445] [General]
Hi.

I wanted to know if having a glTF containing orphan entities (= not parented to root entity = `getParent()` returns `0`) is a normal behavior. Letting glTF having world transformed entities instead of being always parented to root or if it's comming from the `TransformManager` not returning the correct parent.

Thanks

**Comment by dhaval-android** [comment]
Hi, 
Team if possible replay the above case, whats the scenario or fix for the above case, because we are facing a crash in the above scenario.
